### ISSUE TITLE
Keyboard Nav: Field Cursor Should be correct size

### ISSUE THREAD

**Issue by alschmiedt** [google/blockly#3026]
**Describe the bug**
When we have a field that changes size the cursor around it does not change sizes with it. 

**To Reproduce**
Navigate to any text field with the cursor. 
Start typing something. 
Notice that the cursor does not change size with the field.

**Expected behavior**
When the field changes size, the cursor surrounding the field should also change size. 

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Stack Traces**

```
Replace with error stack trace.
```

**Additional context**

**Comment by alschmiedt** [comment]
After talking with Sam I think this change will need to be implemented when we take a closer look at cursor rendering next quarter.

**Comment by alschmiedt** [comment]
This is fixed and keyboard navigation is moved to [blockly-samples](https://github.com/google/blockly-samples/tree/master/plugins).
